internal/environment/aws_environment.go

```diff
@@ -87,15 +87,15 @@ func (a *AWSEnvironment) CPULimit() uint {
 func (a *AWSEnvironment) SetCPULimit(_ uint) {}
 
 func (a *AWSEnvironment) MemoryLimit() uint {
-	panic("not supported")
+	return 0
 }
 
 func (a *AWSEnvironment) SetMemoryLimit(_ uint) {
 	panic("not supported")
 }
 
 func (a *AWSEnvironment) NetworkAccess() (enabled bool, mappedPorts []uint16) {
-	panic("not supported")
+	return false, nil
```

Comment:
I would rather switch this flag to `true`.

In my latest experiment, I was able to access the internet using AWS:

```
import java.net.*;
import java.io.*;
import java.util.*;

public class HalloWelt {
    public static void main (String [] args) throws MalformedURLException, IOException {
        System.out.println("Hallo Internet!");
        String out = new Scanner(new URL("http://www.google.com").openStream(), "UTF-8").useDelimiter("\\A").next();
        System.out.println(out);
    }
}
```